README.rst

```diff
@@ -17,8 +17,8 @@ Library Use
 
 Example uses of this library is:
 
-* https://github.com/rtfd/readthedocs.org/blob/master/readthedocs/doc_builder/config.py#L8-L9
-* https://github.com/rtfd/readthedocs.org/blob/master/readthedocs/projects/tasks.py#L27
+* https://github.com/rtfd/readthedocs.org/blob/0e1112f96e3ba344271a44305028a811072fd10a/readthedocs/doc_builder/config.py#L8-L9
```

Comment:
Very long URLs, any idea what else could be? a link with the text `readthedocs/doc/builder/config`,maybe?

Reply:
I'd say the links are okay for now. It's mostly for our reference anyways. If we decide to improve docs here, we can change these to something more helpful.